app/models/outcome_coverage.rb

```diff
@@ -1,7 +1,10 @@
 class OutcomeCoverage < ActiveRecord::Base
   belongs_to :course
-  belongs_to :outcome
-  belongs_to :subject
+  belongs_to :outcome, required: true
+  belongs_to :subject, required: true
```

Comment:
This is simply a comment/ fun fact that I never knew: Apparently in Rails 5, `required: true` is the default but in Rails 4.x `required: false` was the default, which is why you needed the line `validates, presence: true`. See [here](https://jasoncharnes.com/2016/03/29/rails-belongs_to-required-in-rails-5/) or [here](http://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html).